### PR TITLE
Basic DA-BEAT types' templates for projects

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-5314

Most of the types are in place, BUT we will expand/alter these as we proceed with the frontend since later on we will be in need of diagrams and markdown - so this will be adjusted along the way.

### CODE REVIEW

**Review by torztomasz — APPROVED** [review]

File: packages/frontend2/src/app/(new)/data-availability/_utils/getDaRiskView.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  return bridge.risks.exitWindow
+}
+
+function getAccessibility(bridge: DacBridge | OnChainDaBridge) {
+  if (bridge.type === 'DAC') {
+    return DaAccessabilityRisk.NOT_ENSHRINED
+  }
+
+  return bridge.risks.accessability
+}
```
Comment (torztomasz) [inline packages/frontend2/src/app/(new)/data-availability/_utils/getDaRiskView.ts:88]: include in config

**Review by torztomasz — CHANGES_REQUESTED** [review]

File: packages/config/src/projects/other/da-beat/types/DaBridge.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+     * Description of the data availability bridge
+     */
+    description: string
+
+    /**
+     * The logo of the data availability bridge.
+     * Will be resolved automatically by the frontend.
+     * @override if the logo has different name
+     */
+    logo?: string
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/config/src/projects/other/da-beat/types/DaBridge.ts]: Not now, removed

File: packages/config/src/projects/other/da-beat/types/DaLayer.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+     * A short description of the data availability layer.
+     */
+    description?: string
+
+    /**
+     * The logo of the data availability layer.
+     * Will be resolved automatically by the frontend.
+     * @override if the logo has different name
+     */
+    logo?: string
```
[the remark this answers is not in the document]
Comment (sdlyy) [reply packages/config/src/projects/other/da-beat/types/DaLayer.ts]: Not now, removed

File: packages/frontend2/src/app/(new)/data-availability/_utils/getDaRiskView.ts (hunk 1 above)
Comment (sdlyy) [reply packages/frontend2/src/app/(new)/data-availability/_utils/getDaRiskView.ts]: Fixed and unified

**Review by torztomasz — APPROVED** [review]

**Pull request merged**